WordPress/Classes/Models/KeyringConnection.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+/**
+ KeyringConnection represents a keyring connected to a particular external service.
+ We only rarely need keyring data and we don't really need to persist it. For these
+ reasons KeyringConnection is treated like a model, even though it is not an NSManagedObject,
+ but also treated like it is a Remote Object.
+*/
+@objc public class KeyringConnection : NSObject
+{
+    public var additionalExternalUsers = NSArray()
```

Comment:
Can we specify the kind of entities this fellow will be storing?

Maybe something like.... `= [User]()` ?